e2e/__tests__/__snapshots__/expect-async-matcher.test.js.snap

```diff
@@ -9,9 +9,8 @@ exports[`shows the correct errors in stderr when failing tests 1`] = `
 
   ● fail with expected non promise values
 
-    Error
+    Expected value to have length:
```

Comment:
Maybe this is worse? I dunno.

Reply:
nah, this is better 🙂 more closely matches the sync matcher